superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.jsx

```diff
@@ -161,79 +112,74 @@ export default class SqlEditorLeftBar extends React.PureComponent {
     </IconTooltip>
   );
 
-  render() {
-    const shouldShowReset = window.location.search === '?reset=1';
-    const tableMetaDataHeight = this.props.height - 130; // 130 is the height of the selects above
-    const qe = this.props.queryEditor;
-    return (
-      <div className="SqlEditorLeftBar">
-        <TableSelector
-          database={this.props.database}
-          dbId={qe.dbId}
-          getDbList={this.getDbList}
-          handleError={this.props.actions.addDangerToast}
-          onDbChange={this.onDbChange}
-          onSchemaChange={this.onSchemaChange}
-          onSchemasLoad={this.onSchemasLoad}
-          onTableChange={this.onTableChange}
-          onTablesLoad={this.onTablesLoad}
-          schema={qe.schema}
-          sqlLabMode
-        />
-        <div className="divider" />
-        <StyledScrollbarContainer>
-          <StyledScrollbarContent contentHeight={tableMetaDataHeight}>
-            <Collapse
-              activeKey={this.props.tables
-                .filter(({ expanded }) => expanded)
-                .map(({ id }) => id)}
-              css={theme => css`
-                .ant-collapse-item {
-                  margin-bottom: ${theme.gridUnit * 3}px;
-                }
-                .ant-collapse-header {
-                  padding: 0px !important;
-                  display: flex;
-                  align-items: center;
-                }
-                .ant-collapse-content-box {
-                  padding: 0px ${theme.gridUnit * 4}px 0px 0px !important;
-                }
-                .ant-collapse-arrow {
-                  top: ${theme.gridUnit * 2}px !important;
-                  color: ${theme.colors.primary.dark1} !important;
-                  &: hover {
-                    color: ${theme.colors.primary.dark2} !important;
-                  }
+  const shouldShowReset = window.location.search === '?reset=1';
+  const tableMetaDataHeight = props.height - 130; // 130 is the height of the selects above
+  const qe = props.queryEditor;
+
+  return (
+    <div className="SqlEditorLeftBar">
+      <TableSelector
+        database={props.database}
+        dbId={qe.dbId}
+        getDbList={() => getDbList}
+        handleError={props.actions.addDangerToast}
+        onDbChange={onDbChange}
+        onSchemaChange={onSchemaChange}
+        onSchemasLoad={() => onSchemasLoad}
+        onTableChange={onTableChange}
+        onTablesLoad={() => onTablesLoad}
```

Comment:
same here.. don't need the extra function wrapper. Actually this line and the above will just return the function and not instantiate it.  Let me know if the above recommendation doesn't work.